packages/flutter/test/widgets/routes_test.dart

```diff
@@ -2509,6 +2509,35 @@ void main() {
         moreOrLessEquals(xLocationIntervalTwelve, epsilon: 0.1),
       );
     });
+
+    testWidgets('ModalRoute.isFirstOf only rebuilds when first route state changes', (
```

Comment:
Would you mind adding a similar test for each of the other `of` methods that you added as well?

Reply:
Sure, I will add.